src/Utils.js

```diff
@@ -27,18 +34,94 @@ class Utils {
     return Promise.all(directories.map((directory) => fs.rmdir(directory, { recursive: true })));
   }
 
+
+  /**
+   *
+   *
+   * @static
+   * @param {} data
+   * @param {string} filename:  
+   * @param {object} meta
+   * @returns {Promise}:        A Promise of the pending fs or S3 save
+   * @memberof Utils
+   */
+  static saveFile(data, filename, meta) {
+    const metaType = Utils.identifyFileSystem(meta);
+    const snapshotPath = path.join(meta, filename);
+    if (metaType !== "file") {
+      console.error(`>>> Skipping ${snapshotPath}. Utils.saveFile only supports local filesystems at this time. Message only. No error will be thrown.`);
+      return;
+    }
+    const promise = fs.writeFile(snapshotPath, data);
+    console.log(`>>> Saved ${snapshotPath}.`);
+    return promise;
+  }
+
   /**
    * Returns the mimetype matching the supplied extension. Empty extensions are presumed to come from .html slugs.
    * @static
    * @param {string} ext: The extension, if available, to lookup the mime type for
    * @returns {string}:   The corresponding mimetype, or text/html if no extension was provided.
    * @memberof Utils
    */
-  static getMimeType(path) {
-    const mimeType = mime.getType(path);
+  static getMimeType(aPath) {
+    const mimeType = mime.getType(aPath);
     return mimeType || "text/html";
   }
 
+
+  /**
+   *
+   * Identifies the type of path as filesystem, S3 or HTTP 
```

Comment:
Super nit-picky, however, there is an empty line at the beginning of the comment header.